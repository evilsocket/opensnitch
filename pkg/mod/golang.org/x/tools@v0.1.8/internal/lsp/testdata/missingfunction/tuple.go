package missingfunction

func tuple() {
	undefinedTuple(b()) //@suggestedfix("undefinedTuple", "quickfix")
}

func b() (string, error) {
	return "", nil
}
