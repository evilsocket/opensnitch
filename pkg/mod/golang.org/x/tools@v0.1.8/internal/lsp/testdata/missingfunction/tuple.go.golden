-- suggestedfix_tuple_4_2 --
package missingfunction

func tuple() {
	undefinedTuple(b()) //@suggestedfix("undefinedTuple", "quickfix")
}

func undefinedTuple(s string, err error) {
	panic("unimplemented")
}

func b() (string, error) {
	return "", nil
}

